share/revng/test/tests/analysis/CollectCFG/x86_64/switch-jump-table.S

```diff
@@ -10,12 +10,16 @@ _start:
     ja     end
     jmp    QWORD PTR [rax*8+jumptable]
 one:
+    mov rax, 0x3
     ret
 two:
+    mov rax, 0x5
     ret
 three:
+    mov rax, 0x6
     ret
 end:
+    mov rax, 0x1
```

Comment:
Are you actually using this test case?
Or are you just using those in `for-simplify-switch.yml`?

Reply:
It was a leftover, but we will use this .S file for the test.